printer_mods/cdc/v0_Foldable_Display_Mount/.metadata.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+title: v0 Foldable Display Mount
+
+description: A foldable display mount for v0 that allows the entire printer to fit into my travel backpack.
+
+mod_version: 1
+
+cad:
+- CAD/v0_Foldable_Display_Mount.scad
+
+stl:
+- STL/STL_FILE_1.stl
+- STL/STL_FILE_2.obj
```

Comment:
Looks like you missed removing these two lines from the template

Reply:
Oops, sorry... Thanks a lot for your review!